src/cts/src/TritonCTS.cpp

```diff
@@ -297,6 +297,7 @@ void TritonCTS::writeDataToDb()
 {
   for (TreeBuilder* builder : *builders_) {
     writeClockNetsToDb(builder->getClock());
+    // writeClockNDRsToDb();
```

Comment:
Commented code should be removed, or maybe add an explanation on why it is still there.

Reply:
This code is under active development now.  I plan to add a command line option to enable/disable NDR.

Reply:
Added the following comment:
 // TODO: enable this under command line option
 // writeClockNDRsToDb();
  }